doc/classes/Input.xml

```diff
@@ -309,6 +315,7 @@
 			<param index="0" name="guid" type="String" />
 			<description>
 				Removes all mappings from the internal database that match the given GUID.
+				If you didn't set a GUID fallback mapping (see [method get_fallback_mapping]), then this only affects Android: If the GUID is used as a fallback mapping, the fallback will be invalidated. Use [method get_fallback_mapping] before calling this function to check your current setting. Call [method set_fallback_mapping] to set a new fallback GUID, which must be known (see [method is_joy_known]).
```

Comment:
Not sure if this is absolutely necessary, but either way:
```suggestion
				[b]Note:[/b] If you didn't set a GUID fallback mapping (see [method get_fallback_mapping]), then this method only affects Android: If the GUID is used as a fallback mapping, the fallback will be invalidated. Use [method get_fallback_mapping] before calling this function to check your current setting. Call [method set_fallback_mapping] to set a new fallback GUID, which must be known (see [method is_joy_known]).
```

Reply:
I shortened it. Hope that helps